### ISSUE TITLE
Patterns styles not generated

### ISSUE THREAD

**Issue by marcovanharten-cpi** [chakra-ui/panda#1102]
### Description

When using a pattern function, like `hstack`, the css is not generated. Any other directives like `css` do work however, so I think the panda config file is working.

```ts
// panda.config.ts
import { defineConfig } from '@pandacss/dev'

const config: ReturnType<typeof defineConfig> = defineConfig({
  preflight: true,
  include: ['./src/**/*.{js,jsx,ts,tsx}'],
  exclude: [],
  theme: {
    extend: {},
  },
  outdir: 'styled-system',
})

export default config
```

```css
/* src/global.css */
@layer reset, base, tokens, recipes, utilities;
```

```tsx
// src/app/layout.tsx
import 'src/global.css'
...
```

```tsx
// src/components/navigation.tsx
import { cx } from '@styles/css'
import { hstack } from '@styles/patterns'

export type NavigationProps = { className?: string }

export default function Navigation({ className }: NavigationProps) {
  ...
  return (
    <nav className={cx(hstack({ justify: 'space-between' }), className)}>
      ...
    </nav>
  )
}
```

Expected
---
The styles to be included

Reality
---
The classes are included but the styles are not (only for pattern functions)
![image](https://github.com/chakra-ui/panda/assets/114489263/f6e542b1-a87c-40a8-988d-5058b8c90e2c)
![image](https://github.com/chakra-ui/panda/assets/114489263/b7894ce9-c7c9-4962-aff4-edd681750219)

### Link to Reproduction

https://cannot-create-a-next-app-router-app-in-stackblitz.com

### JS Framework

NextJS (TS)

### Panda CSS Version

0.7.0

### Browser

Google Chrome 115

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

I have a suspicion that the files are only being watched for the `css` function and nothing else.

**Comment by astahmer** [comment]
Hey, it's already fixed but not yet merged & published on npm
it'll be available once https://github.com/chakra-ui/panda/pull/1055 gets merged
and [this](https://github.com/chakra-ui/panda/commit/be0ad578156d1bed81ab26be770569674b56fb3d) was the PR that fixed this issue

**Comment by marcovanharten-cpi** [comment]
> Hey, it's already fixed but not yet merged & published on npm it'll be available once #1055 gets merged and [this](https://github.com/chakra-ui/panda/commit/be0ad578156d1bed81ab26be770569674b56fb3d) was the PR that fixed this issue

Thanks for the info. I'll keep track of that PR and close this Issue. Do you have any ETA on the release?

**Comment by astahmer** [comment]
pretty soon I guess ? 😅  can't be more precise
